core/scripts/vrfv2plus/testnet/proofs.go

```diff
@@ -49,7 +49,7 @@ var rcTemplate = `{
 	callbackGasLimit: %d,
 	numWords: %d,
 	sender: %s,
-	nativePayment: %t
+	nativePayment: %s
```

Comment:
not sure why we had `%t` here initially. makes more sense to print hex-encoded string of the extraArgs instead of the type. this should be renamed to `extraArgs` actually.

Reply:
Isn't lowercase %t boolean in Go? Or am I mixing that up?

Reply:
oh right. yeah it was trying to display bytes as bool